### ISSUE TITLE
Both Hands interaction - no values yet

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#110]
Currently we have no description for the both hands interaction.

Can we just implement the current `aria-valuetext` values for that interaction. Is it possible?

**Comment by terracoda** [comment]
If possible, please do it. Otherwise, please let us know.

**Comment by zepumph** [comment]
aria-valuetext didn't seem to work for me. I think this is because it has a `role="application"` on it after trying a few options there, I thought that instead the best way forward, at least for the first round of interviews, is to just use aria-live alerts. 

To review:
* I made up an alert pattern that used both left and right positions. Would you rather a different text?
* We alert both each time right now because the "both hands listener" doesn't know how to differentiate between when the left changes and when the right changes. 
* I used our `Utterance` tech that won't alert too often, and although it may need tweaking, it sounds pretty good to me. Please let me know if you want the repetition tweaked in any way.


Thanks!

**Comment by Matthew-Moore240** [comment]
The main issue I see with this right now is they get no info on where they are in relation to the ratio. Its possible this is fine for now but i would like @terracoda and @emily-phet opinions on this.

**Comment by zepumph** [comment]
Oh, right! I should add that too. I put it after the position. Right now it is obvious that this is way too verbose, but it is at least a start, and better than not having this information. Let me know if you want it tweaked.

**Comment by emily-phet** [comment]
Yes - it is key that the learner hear text description that let's them know they have found the success point, so they have confirmation of that and can understand what the success ding sound means. Otherwise, we can ask them to look for the ratio, but they never know if they're there. 

@Matthew-Moore240 can you listen to what @zepumph has in place now and see if you think it needs.

**Comment by emily-phet** [comment]
Whoops - I may have misinterpreted, I hadn't read the entire thread. Ignore if it seems off-base. I'll take a full read through later on.

**Comment by terracoda** [comment]
Apologies for th econfusion, @zepumph, I meant use the content from the aria-valuetext. Of course the attribute aria-valuetext will only work on input type range, (i.e. a slider).

**Comment by terracoda** [comment]
This is very buggy on VoiceOver.

Sometimes I hear nothing.
Sometimes I hear ratio.
Haven't heard positions yet.

I think the object response (position value) and context response need to be put into one response. Firing 2 responses in quick succession might interrupt what is happening too much.

![Screen Shot 2020-07-31 at 2 22 22 PM](https://user-images.githubusercontent.com/1902116/89058098-499f6180-d339-11ea-96f7-fff974279f76.png)

**Comment by jessegreenberg** [comment]
Self-assigning to test on VoiceOver.

**Comment by jessegreenberg** [comment]
After the above commit it is working much better for me. It is ready for review again.

**Comment by terracoda** [comment]
@jessegreenberg, need dev link as soon as possible :-)

**Comment by emily-phet** [comment]
@terracoda - @jessegreenberg put a link to a dev version on Slack, the RaP channel Friday afternoon.

**Comment by emily-phet** [comment]
Closing this issue - I think we have a plan for a next approach to the content that is displayed for the "both hands" scenario, and will create an issue focused solely on that approach.
